### PR TITLE
Auth screens UI

### LINKED ISSUE #92: Figma design: Auth screen
### OD Hack

This issue is part of ODHack 4.0:

description - https://x.com/OnlyDust_com/status/1791035111992934563
guidelines - https://onlydust.notion.site/ODHack-Common-Guidelines-Project-leads-5e21836f85ad493f9ca429770af0ad40

###  Figma for the new screens

Create new screens using the [Auth module](https://github.com/keep-starknet-strange/joyboy/blob/main/JoyboyCommunity/src/modules/auth/index.tsx) in [screens folder](https://github.com/keep-starknet-strange/joyboy/tree/main/JoyboyCommunity/src/screens/Auth) based on the Figma designs created by @Israelrex9 These new screens should match the designs 1:1

No need for login and create account functionalities, only creating the screens based on the designs is sufficient.

Figma Designs: https://www.figma.com/design/NIJ68MV1AeI8y283woXiM0/JoyBoy?node-id=2-274&t=vCPEMKXnFXuQ3fVJ-0

Required screens are:
- Login
- Create account
- Save your keys

Please battle test it before doing a PR.

### Communication

https://t.me/JoyboyStarknet/949

May the Joy be with you!

### PR DESCRIPTION
**Pull request by princeibs**
Closes #92

### CODE REVIEW

File: JoyboyCommunity/assets/svgs/svgComponents/DoubleCopy.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import Svg, {SvgProps, Path} from 'react-native-svg';
+const DoubleCopyIcon = (props: SvgProps) => (
+  <Svg width={24} height={24} fill="none" {...props} viewBox="0 0 24 24">
```
Comment (ugur-eren) [inline JoyboyCommunity/assets/svgs/svgComponents/DoubleCopy.tsx:4]: Icons moved to the [icons.tsx](https://github.com/keep-starknet-strange/joyboy/blob/be01de1509e512a128715d04cfc5b90beee49fc1/JoyboyCommunity/src/assets/icons.tsx) file, can you move the icons to there?

File: JoyboyCommunity/assets/svgs/svgComponents/Info.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import Svg, {SvgProps, Path} from 'react-native-svg';
+const InfoIcon = (props: SvgProps) => (
+  <Svg width={18} height={18} fill="none" {...props} viewBox="0 0 18 18">
+    <Path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
```
Comment (ugur-eren) [inline JoyboyCommunity/assets/svgs/svgComponents/Info.tsx:7]: Svg attributes should be camelCased, for example `fill-rule` should be `fillRule`

**Review by ugur-eren — CHANGES_REQUESTED** [review]
Hi, I've requested some changes regarding the SVGs. Other than that looks great! Thank you for the work.

**Comment by princeibs** [comment]
All issues resolved ✅
@ugur-eren

**Review by ugur-eren — APPROVED** [review]
Thanks for the great work! LGTM

**Pull request merged**